uiServices/src/services.go

```diff
@@ -28,7 +28,7 @@ func validateFileNameAndHeader(rw http.ResponseWriter, req *http.Request, header
 
 	if len(name) <= 1 {
 		logrus.Error("File Not Found")
```

Comment:
Why is a single character incorrect for a file name? is A.xml an invalid filename?